src/cdef.rs

```diff
@@ -128,10 +127,9 @@ fn constrain(diff: i32, threshold: i32, damping: i32) -> i32 {
 // of the 2-pixel padding around the block, not the block itself.
 // The destination is unpadded.
 unsafe fn cdef_filter_block<T: Pixel>(
-  dst: *mut T, dstride: isize, input: *const T, istride: isize, pri_strength: i32,
+  dst: *mut T, dstride: isize, input: *const u16, istride: isize, pri_strength: i32,
```

Comment:
Oh cool, I wondered what to do with these `CDEF_VERY_LARGE` (=30000) when `T == u8`.
If the input [is always `u16`](https://aomedia.googlesource.com/aom/+/b6f1767eedbaddeb1ff5aa409a710ef61078640e/av1/common/cdef_block.h#45), problem solved :+1: